src/mango/test/19-find-conflicts.py

```diff
@@ -25,9 +26,25 @@ def setUp(self):
         self.db.save_docs_with_conflicts(copy.deepcopy(CONFLICT))
 
     def test_retrieve_conflicts(self):
-        self.db.create_index(["_conflicts"])
+        self.db.create_index(["_conflicts"], wait_for_built_index=False)
         result = self.db.find({"_conflicts": {"$exists": True}}, conflicts=True)
         self.assertEqual(
             result[0]["_conflicts"][0], "1-23202479633c2b380f79507a776743d5"
         )
         self.assertEqual(result[0]["_rev"], "1-3975759ccff3842adf690a5c10caee42")
+
+    @unittest.skip("CouchDB does not support interactive conflict indexing")
```

Comment:
Why would you add a test to just disable it?

Reply:
As a reminder that we don't yet support it for interactive updater.